ext/lib/crypto/tinycrypt/include/tinycrypt/ecc_platform_specific.h

```diff
@@ -58,6 +58,10 @@
 #ifndef __UECC_PLATFORM_SPECIFIC_H_
 #define __UECC_PLATFORM_SPECIFIC_H_
 
+#ifdef __cplusplus
```

Comment:
please split this into its own commit and send tinycrypt patches to https://github.com/01org/tinycrypt/pulls